src/datatable.js

```diff
@@ -249,7 +297,7 @@ var dataTable = {
             })
             .fail(function(response, option, err) {
                 var requestErr = httpErrorParser.parse(response, option, err);
-                logger.error(requestErr.message);
+                // logger.error(requestErr.message);
```

Comment:
Is it wanted or is it a forgotten debug action?
Better to avoid having dead code, so if this was wanted, please remove the line. Otherwise please restore it :)

Reply:
Uncomment